static/js/home/snatch-selection.js

```diff
@@ -1,20 +1,20 @@
 MEDUSA.home.snatchSelection = function() {
     $('.imdbPlot').on('click', function() {
```

Comment:
This should be moved into an init function or something similar as it's used on multiple pages. For another PR though.